README.md

```diff
@@ -42,10 +42,10 @@ optional arguments:
   -h, --help     show this help message and exit
   -v, --verbose  Verbose level... repeat up to three times.
   -s, --silent   Silent - only errors will be printed.
+  --version      show program's version number and exit
```

Comment:
even better would be `-v --version` and verbose with `-d` .. but I'm fine with this as well..